src/registrar/tests/test_models_domain.py

```diff
@@ -964,6 +978,22 @@ def test_is_disclosed_on_security_contact(self):
         # Confirm that we are getting the desired email
         self.assertEqual(domain.security_contact.email, expectedSecContact.email)
 
+    def test_disclose_security_emails(self):
+        """
+        Tests that command disclose_security_emails runs successfully with
+        appropriate EPP calll to UpdateContact.
+        """
+        domain, _ = Domain.objects.get_or_create(name="igorville.gov")
+        expectedSecContact = PublicContact.get_default_security()
+        expectedSecContact.domain = domain
+        expectedSecContact.email = "123@mail.gov"
+        domain.security_contact = expectedSecContact
+        self.run_disclose_security_emails()
```

Comment:
this call doesn't get tested at all. The line 990 above already sent a create/update contact message so the assert would already return true. I would check that the call count is two on the mock, at the very least.

Reply:
Updated tests to verify that calling disclose specifically makes an EPP call to UpdateContact instead to be more specific to the script